config/locales/en.yml

```diff
@@ -35,6 +35,7 @@ en:
       empty_file_warning: "%{file_name} is empty"
       external_submit_only: MarkUs is only accepting external submits
       file_conflicts: 'Your changes have not been made.  The following file conflicts were detected:'
+      file_extension_mismatch: The uploaded file doesn't match its file extension %{extension}.
```

Comment:
Reword to "The contents of the uploaded file do not match the file extension %{extension}".